sdk/communication/azure-communication-identity/tests.yml

```diff
@@ -19,3 +19,6 @@ stages:
               - $(sub-config-communication-int-test-resources-common)
               - $(sub-config-communication-int-test-resources-python)
       Clouds: Public,Int
+      EnvVars:
+        AZURE_SKIP_LIVE_RECORDING: 'True'
+        AZURE_TEST_RUN_LIVE: 'true'
```

Comment:
What do these settings do exactly? (link to some docs would help)

Reply:
@petrsvihlik these lines enable proxy on live pipelines
You can check the migration guide for more information: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/test_proxy_migration_guide.md#live-test-pipelines

Reply:
thank you!